tests/plots/test_beeswarm.py

```diff
@@ -3,6 +3,21 @@
 import pytest
 
 import shap
+from shap.plots.colors import (
+    blue_rgb,
+    gray_rgb,
+    light_blue_rgb,
+    light_red_rgb,
+    red_blue,
+    red_blue_circle,
+    red_blue_no_bounds,
+    red_blue_transparent,
+    red_rgb,
+    red_transparent_blue,
+    red_white_blue,
+    transparent_blue,
+    transparent_red,
+)
```

Comment:
Nitpick: it might be cleaner to keep these objects in their module namespace, rather than import each one individually:

```python
from shap.plots import colors
...
...
@pytest.mark.parametrize(
    "color",
    [
        colors.blue_rgb,
        colors.gray_rgb,
        ...
```